packages/lib/src/Miew.js

```diff
@@ -2888,24 +2897,30 @@ Miew.prototype._onKeyDown = function (event) {
       }
       break;
     case 'S'.charCodeAt(0):
-      event.preventDefault();
-      event.stopPropagation();
+      if (!['TEXTAREA', 'INPUT'].includes(document.activeElement?.tagName)) {
```

Comment:
I wonder if there are other tag names to support. Or even there could be an arbitrary DIV element (a React component) waiting for the S key and this solution wouldn't help. This fix might be a short-term solution if `miew.enableHotKeys(false)` doesn't work for you.

The long-term solution I see is:
- move all key handlers to the demo apps providing proper public methods in the library;
- disable the `S` key completely as it looks more like a debugging key;

However, it is just my quick glance with a dump of current thoughts.